examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/README.md

```diff
@@ -0,0 +1,113 @@
+# Mina zkApp: Zkapp Offchain Storage
+
+An experimental library for storing zkApp data offchain. This project is currently under development and should not be used in production.
+
+## Usage
+
+### Dependencies:
+
+Ensure all required modules are installed. To do so, run the following command:
+
+```bash
+npm install
+```
+
+### Running the Example
+
+#### Blockchain Mode
+
+Toggle the `useLocalBlockchain`` variable to switch between local testing and real-world usage.
```

Comment:
It might be useful to call out that the the `useLocalBlockchain` variable is in the `main.ts` file